inspector/src/main/java/io/sweers/inspector/Inspector.java

```diff
@@ -15,7 +15,8 @@
  * Validates objects.
  */
 public final class Inspector {
-  static final List<Validator.Factory> BUILT_IN_FACTORIES = new ArrayList<>(5);
+  private static final List<Validator.Factory> BUILT_IN_FACTORIES = new ArrayList<>(5);
+  private static final ThreadLocal<List<DeferredAdapter<?>>> REENTRANT_CALLS = new ThreadLocal<>();
```

Comment:
btw, I know Error Prone complains here, but it's wrong. Using `remove` below makes using the original instance field the correct behavior.

You technically don't want to share your cache globally, across all instances of the Inspector client, I'm guessing.

/unsolicited-comments

Reply:
unsolicited comments welcome :). Thanks for pointing this out, I admittedly just blindly followed error-prone